packages/reports/reports.hjson

```diff
@@ -0,0 +1,212 @@
+{
+  "reports": [
+    {
+    "id": "8",
+    "config": {
+      "fetch": [
+        {
+          "fetchAnalytics": {
+            "dataElementCodes": [
+              "BCD46",
+              "BCD47",
+              "BCD48",
+              "BCD48a",
+              "BCD48b",
+              "BCD49",
+              "BCD50",
+              "BCD51",
+              "BCD52",
+              "BCD53",
+              "BCD54",
+              "BCD55"
+            ],
+            "aggregationType": "RAW",
+            "dataServices": [
+              {
+                "isDataRegional": true
+              }
+            ]
+          }
+        }
+      ],
+      "transform": [
+        {
+          "transform": "select",
+          "$.dataElement": "$.value",
```

Comment:
Clever! So `$` is the row, and `$.` is `row.`?

Reply:
Exactly, `mathjs` lets you add objects to the expression scope, and you can reference them with standard JS notation.
eg. `$.BCD1 or $['BCD1']`